src/inactive-regions.ts

```diff
@@ -90,6 +85,24 @@ export class InactiveRegionsFeature implements vscodelc.StaticFeature {
     this.applyHighlights(filePath);
   }
 
+  updateDeclaratedType() {
+    if (this.decorationType) {
```

Comment:
nit: this can be written slightly more concisely as `this.decorationType?.dispose();`